lib/mi_bridges/driver/household_member_questions_page.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class HouseholdMemberQuestionsPage < BasePage
+      delegate(
+        :anyone_disabled?,
+        :primary_member,
+        to: :snap_application,
+      )
+
+      delegate :first_name, to: :primary_member
+
+      def setup; end
+
+      def fill_in_required_fields
+        check_blindness_or_disability
+        check_drug_felonies
+        check_probation_or_parole
+        check_other_food_assistence_benefits
+        check_ssi_benefit
+        check_cash_benefit_from_mdhhs
+        check_refugee_asylee_or_victim
+      end
+
+      def continue
+        click_on "Next"
+      end
+
+      def check_blindness_or_disability
+        check_in_section(
+          "starBlindnessorDisability",
```

Comment:
interesting how many (or all?) of these names start with `star` -- did you ever consider moving that part of the string into the `check_in_section` method itself?